compiler.js

```diff
@@ -194,8 +194,8 @@
             start: /^@start (.*)$/,
             attributes: /^@set (.*)$/,
             unset: /^@unset (.*)$/,
-            inc: /^@inc (.*)$/,
-            dec: /^@dec (.*)$/,
+            inc: /^@inc (.*)( )(.\d*)$/,
```

Comment:
This is going to break the existing inc/dec syntax. If no number is specified, it should still inc/dec by 1.